src/main/scala/catslib/Apply.scala

```diff
@@ -48,7 +46,7 @@ object ApplySection extends FlatSpec with Matchers with org.scalaexercises.defin
     * Since `Apply` extends `Functor`, we can use the `map` method from `Functor`:
     */
   def applyExtendsFunctor(res0: Option[String], res1: Option[Int], res2: Option[Int]) = {
-    import cats.std.all._
+    import cats.implicits._
```

Comment:
This PR has focused in the upgrading process. Maybe we could align the current cats documentation (like http://typelevel.org/cats/tut/apply.html) with the current implementation in scala-exercises in a different ticket: https://github.com/scala-exercises/exercises-cats/issues/46